dispatch/static/manager/src/js/components/inputs/SectionSelectInput.js

```diff
@@ -17,17 +17,40 @@ class SectionSelectInputComponent extends React.Component {
     this.props.listSections(this.props.token, queryObj)
   }
 
+  getCurrentFilter(selected) {
+    if (this.props.entities.sections[selected]) {
+      return this.props.entities.sections[selected].name
+    }
+
+    return 'Filter by Section'
+  }
```

Comment:
Same here -- is this still needed?

Reply:
Right those shouldn't be needed anymore. I'll remove them.